src/app.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+
+def create_app(config_file):
+    app = Flask(__name__)
+    app.config.from_pyfile(config_file)
+
+    from models import db, migrate
```

Comment:
Cargo culting from http://flask.pocoo.org/docs/0.12/patterns/appfactories/. Happy to change it if you're bothered.